cmd/boltzcli/commands.go

```diff
@@ -193,57 +196,75 @@ func listSwaps(ctx *cli.Context) error {
 		headerFmt := color.New(color.FgGreen, color.Underline).SprintfFunc()
 		columnFmt := color.New(color.FgYellow).SprintfFunc()
 
-		if len(list.Swaps) == 0 && len(list.ReverseSwaps) == 0 && len(list.ChainSwaps) == 0 {
-			fmt.Println("No swaps found")
-			return nil
-		}
+		if unify {
```

Comment:
This is always going to be true. Why keep the if?

Reply:
Initially thought that i would make a flag for it. but i dont think thats necessary